leshan-core/src/main/java/org/eclipse/leshan/core/json/jackson/JsonArrayEntrySerDes.java

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Sierra Wireless and others.
```

Comment:
Should this be a bit more recent? (same for others)

Reply:
I will fix it (bad copy/paste)